regen-network/regen-web
# feat: APP-183 attach signature to data post + APP-240

## Description

https://regennetwork.atlassian.net/browse/APP-183?atlOrigin=eyJpIjoiYTczMWFmNjQ1NjhkNGU2Y2FiYTk2NzYzM2E5YWJkMjIiLCJwIjoiaiJ9

https://regennetwork.atlassian.net/browse/APP-240?atlOrigin=eyJpIjoiZmNkYmEwMTIwNjM0NGM5MjlhZjhmNGUxN2JjZmYwMzMiLCJwIjoiaiJ9



---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...


### How to test



From https://deploy-preview-2423--regen-marketplace.netlify.app/
1. Log in with a web2 account and create a post, you should see anchoring hash in the success modal
2. Log in with a web3 account and create a post. After entering post data, you should see a modal to sign the post or skip this step. Test both options. If signing a post successfully, you'll see the attesting hash in the success modal too.

The button at the bottom of the success modal now goes to the newly created post page.

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...
